src/ensureCloudResourceLocation.ts

```diff
@@ -0,0 +1,20 @@
+import * as FirebaseError from "./error";
+
+/**
+ * Simple helper function that returns an error with a helpful
+ * message on event of cloud resource location that is not set.
+ * This was made into its own file because this error gets thrown
+ * in several places: in init for Firestore, Storage, and for scheduled
+ * function deployments.
+ * @param location cloud resource location, like "us-central1"
+ * @throws { FirebaseError } if location is not set
+ */
+export function ensureLocationSet(location: string | undefined, feature: string): void {
```

Comment:
```suggestion
export function ensureLocationSet(location?: string, feature: string): void {
```

Reply:
Actually this was a hack I had to put in because I wanted to use this in the scheduled function deploy flow but it just didn't make sense so I reverted it. Changing this back to `location: string`